content/en/tracing/setup/dotnet-framework.md

```diff
@@ -64,26 +64,47 @@ If your application runs in IIS, skip the rest of this section.
 
 For Windows applications **not** running in IIS, set these two environment variables before starting your application to enable automatic instrumentation:
 
+
+**Note:** The .NET runtime tries to load a profiler into _any_ .NET process that is started with these environment variables are set. You should limit instrumentation only to the applications that need to be traced. **We do not recommend setting these environment variables globally as this causes _all_ .NET processes on the host to load the profiler.**
+
 | Name                   | Value                                    |
 | ---------------------- | ---------------------------------------- |
 | `COR_ENABLE_PROFILING` | `1`                                      |
 | `COR_PROFILER`         | `{846F5F1C-F9AE-4B07-969E-05C26BC060D8}` |
 
-For example, to set the environment variables from a batch file before starting your application:
+#### Windows Services
+
+To automatically instrument a Windows Service, set the required environment variables using a multi-string value in the Windows Registry:
+
+- Key: HKLM\System\CurrentControlSet\Services\<Service Name>
+- Value Name: Environment
+- Value Data:
+CORECLR_ENABLE_PROFILING=1
+CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+
+This can be done either through the Registry Editor as in the image below, or through a PowerShell snippet:
+
+{{< img src="tracing/setup/dotnet/RegistryEditor.png" alt="Registry Editor"  >}}
+
+{{< code-block lang="dotnet" filename="PowerShell" >}}
```

Comment:
The correct `lang` is going to be either `ps` or `powershell`. I'm not sure which one, so let's try this.
```suggestion
{{< code-block lang="ps" filename="PowerShell" >}}
```